### PR TITLE
push back eventing and networking update

### PR DESCRIPTION
**Pull request by lionelvillard**
Due to KnativeCon.

### CODE REVIEW

File: mechanics/TOC-REVIEW.schedule [hunk 1]
```diff
@@ -18,13 +18,14 @@
...
+# 2022-10-27T08:30:00-07:00 | Networking WG
+2022-11-03T08:30:00-07:00 | Eventing WG
+2022-11-10T08:30:00-07:00 | Networking WG
+2022-11-17T08:30:00-07:00 | DUX WG
+2022-11-24T08:30:00-07:00 | Security WG
+2022-12-01T08:30:00-07:00 | Productivity WG
+2022-12-08T08:30:00-07:00 | Client WG
+2022-12-15T08:30:00-07:00 | Serving API WG
+2022-12-22T08:30:00-07:00 | Operations WG
+2022-12-29T08:30:00-07:00 | Functions WG
```
Comment (psschwei) [inline mechanics/TOC-REVIEW.schedule:31]: FYI @houshengbo @lance @salaboy

**Review by psschwei — COMMENTED** [review]
/lgtm
/approve

File: mechanics/TOC-REVIEW.schedule (hunk 1 above)
Comment (lance) [reply mechanics/TOC-REVIEW.schedule:31]: @psschwei thanks for the heads up. I am pretty sure I will be on PTO that day (Red Hat is soft-closed for business between Christmas and New Years. @salaboy what about you?

Comment (psschwei) [reply mechanics/TOC-REVIEW.schedule:31]: I imagine there's a few others who will also be on PTO at the end of the year (I know of at least one ;) ) so feel free to bump to after the new year (same for you @houshengbo )

**Pull request merged**